src/java/org/apache/cassandra/schema/SchemaConstants.java

```diff
@@ -72,9 +74,31 @@ public final class SchemaConstants
 
     public static final List<String> LEGACY_AUTH_TABLES = Arrays.asList("credentials", "users", "permissions");
 
-    public static boolean isValidName(String name)
+    public static boolean isSafeLengthForFilename(String name)
```

Comment:
Why change the name of this public API?
I feel the new name is worse and more confusing and if I didn't know the context of this PR, I'd have trouble figuring out what the author meant by writing "Filename" here. 

The original method name was fine.